x/mint/keeper/hooks_test.go

```diff
@@ -410,25 +408,25 @@ func TestAfterEpochEnd_FirstYearThirdening_RealParameters(t *testing.T) {
 	actualTotalProvisionedSupply := app.BankKeeper.GetSupplyWithOffset(ctx, mintDenom).Amount.ToDec()
 
 	// 299_999_999_999_999.999999999999999705 == 299_999_999_997_380 + 2555 + 64.999999999999999705
-	require.Equal(t, expectedTotalProvisionedSupply, actualTotalProvisionedSupply.Add(devRewardsDelta).Add(epochProvisionsDelta))
+	suite.Require().Equal(expectedTotalProvisionedSupply, actualTotalProvisionedSupply.Add(devRewardsDelta).Add(epochProvisionsDelta))
 
 	// This end of epoch should trigger thirdening. It will utilize the updated
 	// (reduced) provisions.
 	app.MintKeeper.AfterEpochEnd(ctx, epochIdentifier, thirdeningEpochNum)
 
-	require.Equal(t, thirdeningEpochNum, app.MintKeeper.GetLastReductionEpochNum(ctx))
+	suite.Require().Equal(thirdeningEpochNum, app.MintKeeper.GetLastReductionEpochNum(ctx))
 
 	expectedThirdenedProvisions := mintParams.ReductionFactor.Mul(genesisEpochProvisionsDec)
 	// Sanity check with the actual value on mainnet.
-	require.Equal(t, mainnetThirdenedProvisions, expectedThirdenedProvisions.String())
-	require.Equal(t, expectedThirdenedProvisions, app.MintKeeper.GetMinter(ctx).EpochProvisions)
+	suite.Require().Equal(mainnetThirdenedProvisions, expectedThirdenedProvisions.String())
+	suite.Require().Equal(expectedThirdenedProvisions, app.MintKeeper.GetMinter(ctx).EpochProvisions)
 }
 
-func setupGaugeForLPIncentives(t *testing.T, app *osmoapp.OsmosisApp, ctx sdk.Context) {
+func (suite *KeeperTestSuite) setupGaugeForLPIncentives(app *osmoapp.OsmosisApp, ctx sdk.Context) {
```

Comment:
nit: we should not need to make `setupGaugeForLPIncentives` a method of the keeper suite. We can pass it t as an argument by doing `suite.T()`.

I'm suggesting this for consistency with other tests